.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules
 download
 .vscode
 yarn.lock
+package-lock.json
```

Comment:
Isn't the recommendation to add this file to the repo, see e.g. https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json ?

Reply:
Thanks for the doc. I also noticed that after we added `package-lock.json`, we should use `npm ci` instead `npm install`.